bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
+
+from bankroll.model import Position, Trade
+
+ConvertibleModel = Union[Position, Trade]
+ColumnFunctions = List[Tuple[str, Callable[[Any], Any]]]
+
+
+def dataframeForModelObjects(
+    items: Sequence[ConvertibleModel]
+) -> Optional[pd.DataFrame]:
+    df = None
+
+    if module_exists("pandas"):
```

Comment:
If you set `pandas = None` like I suggested above, this can just be written as:

```suggestion
    if pandas:
```